modules/arm_plugin/src/arm_converter/arm_converter_interpolate.cpp

```diff
@@ -56,46 +65,67 @@ void wrap_interpolate(const T* input_data,
 
     pad_input_data(data_ptr, padded_data_ptr, type_size, input_shape, padded_shape, pads_begin);
 
+    std::vector<int64_t> axes_vec;
+    if (axes != nullptr) {
+        auto axes_size = ngraph::shape_size(axes_shape);
+        axes_vec.assign(axes, axes + axes_size);
+    } else {
+        get_default_axes(axes_vec, input_partial_shape);
+    }
+
     auto scales_size = ngraph::shape_size(scales_shape);
     std::vector<float> scales_vec;
     if (attrs.shape_calculation_mode == ngraph::op::v4::Interpolate::ShapeCalcMode::SIZES) {
         for (size_t i = 0; i < scales_size; i++) {
-            auto axis = axes[i];
+            size_t axis = axes_vec[i];
             scales_vec.push_back(static_cast<float>(out_shape[axis]) / padded_shape[axis]);
         }
     } else {
         scales_vec = std::vector<float>(scales, scales + scales_size);
     }
 
-    auto axes_size = ngraph::shape_size(axes_shape);
-    std::vector<int64_t> axes_vec(axes, axes + axes_size);
     ngraph::runtime::reference::interpolate<T>(reinterpret_cast<T*>(padded_data_ptr),
-                                                padded_shape,
-                                                scales_vec,
-                                                axes_vec,
-                                                out,
-                                                out_shape,
-                                                attrs);
+                                               padded_shape,
+                                               scales_vec,
+                                               axes_vec,
+                                               out,
+                                               out_shape,
+                                               attrs);
 }
 
 template<> Converter::Conversion::Ptr Converter::Convert(const opset::Interpolate& node) {
     auto make = [&] (auto refFunction) {
+    if (node.get_input_size() == 4) {
         return this->MakeConversion(refFunction,
                                     node.input(0),
                                     node.get_input_shape(0),
+                                    node.get_input_partial_shape(0),
```

Comment:
`get_input_shape()` infers `Shape` from partial shape if it's static and throw exception otherwise. Usually it's safer to call for partial shape, but we anyway use static shape already so call for partial shape is redundant